governance.md

```diff
@@ -59,260 +70,185 @@ The foundations of Project governance are:
+The Steering Committee may not override a PDEP.  When the Steering Committee meets to
+discuss an issue, the members of the Steering Committee are responsible for soliciting
+input from members of the relevant Teams.
+
+The Steering Committee may create a working group to consider changes to the governance
+model described in this document, including the creation of new Teams that support the
+goals of The Project. Changes to the governance model will require a voting process and
+approval equivalent to the process described in PDEP-1, with the exception that all
+Stewards are eligible to vote.  
+
+The Steering Committee may appoint temporary working groups to work on issues, such as
+governance, that fall outside the scope of responsibilities for the existing Teams.
+
+The ideal composition of the Steering Committee consists of 5 people.  The Steering
+Committee will be chosen via an approval voting process from a slate of nominees,
+meaning that each person who is eligible to vote may vote for more than one candidate
+from the slate.  All members of each Team, except the Code of Conduct Team, are 
+eligible to vote.  
+
+### Initial Election
+
+The initial slate for the Steering Committee election will consist of 5 or more
+candidates who volunteer to be on the slate of candidates for the initial Steering
+Committee.  For the initial election, the members of the previous _pandas_ core team
+will vote via an approval voting process.  The top 5 candidates receiving votes
+will then become members of the initial Steering Committee. 
+
+### Subsequent Elections  
+
+Subsequent changes to the membership of the Steering Committee will occur towards the
+end of each calendar year.  By October 31 of each year, each member of the Steering
+Committee will be asked if they would like to continue in that role. If any member
+decides to step down from the Steering Committee, new volunteers will be solicited from
+the group of stewards by the Steering Committee.  Those new volunteers, along with any
+current Steering Committee members who wish to remain on the Steering Committee,
+will then be on a
+slate for an election that will occur by November 15 of that year.  For each year after
+the initial election, the current Steering Committee members plus any new volunteers on
+the slate will be elected via approval voting, with the top 5 candidates elected as
+Steering Committee members for the subsequent year.
+
+The terms of each member of the Steering Committee are from January
+1 to December 31 of each calendar year, and Steering Committee members may serve for
+any number of multiple terms, provided that they are re-elected in each annual election.
+
+
+## Teams
+
+Each Team has defined responsibilities for different aspects of the project.  As a
+general rule, an Individual Contributor can be nominated by a member of a Team to become
+a member of that Team, and the Team must unanimously agree to admitting that person to
```

Comment:
I think requiring unanimous here could be problematic if someone already on the team steps away from the project or is unable to vote for whatever reason. I think we should have a 2 week-1 month timeframe for a team member to object, otherwise any non-votes are not counted

Reply:
I think the idea was to mean "unanimous consent", or like "someone proposes it, and if there are no objections, we assume everyone agrees", not that actually every individual has to explicitly approve.

Reply:
discussed on 6/5/24.  Removed the word "unanimous" and based decisions on lack of objections from team members.